pydy/system.py

```diff
@@ -545,7 +578,15 @@ def _Kane_constant_symbols(self):
         TODO temporary.
 
         """
-        constants = list(self.eom_method._find_othersymbols(
-            *self._Kane_inlist_insyms()))
-        constants.remove(sym.symbols('t'))
+        from_eoms, from_sym_lists = self._Kane_inlist_insyms()
+        if cmp(parse_version(SYMPY_VERSION), parse_version('0.7.6')) > -1:
```

Comment:
It's not so bad, it just could be cleaner. Perhaps you could do something like this, if you wanted:

```
def newer_than(version):
    return cmp(parse_version(SYMPY_VERSION), parse_version(version)) > -1
```

Then this line would be:

```
if newer_than('0.7.6'):
    stuff...
```